### ISSUE TITLE
FormBuilderDateTimePicker throws Unhandled Exception: A FocusNode was used after being disposed.

### ISSUE THREAD

**Issue by Orlyga01** [flutter-form-builder-ecosystem/flutter_form_builder#905]
Hi,
I have a widget that holds a form.
In this form I have a FormBuilderDateTimePicker
This form is part of an "OnBoarding" page
on each step, another form is being displayed, and each click on "Next" button is triggering seState, that replaces the current form to display.
When clicking (focusing) on FormBuilderDateTimePicker, this error is displayed:
```

 Unhandled Exception: A FocusNode was used after being disposed.
: Once you have called dispose() on a FocusNode, it can no longer be used.
```
This is my class

```class BabyForm extends StatefulWidget {


  GlobalKey<FormBuilderState> babyFormKey;
  BabyForm(
      {Key? key,
 
      this.babyFormValues})
      : super(key: key);

  @override
  _BabyFormState createState() => _BabyFormState();
}

class _BabyFormState extends State<BabyForm> {
 Widget build(BuildContext context) {
    return SingleChildScrollView(
      physics: ClampingScrollPhysics(),
      child: Container(
          child: FormBuilder(
        key: widget.babyFormKey,
        child: Column(
          children: [
            Text(("Your baby").tr(), style: BeStyle.H2),
            FormBuilderDateTimePicker(
              name: 'timeOfBirth',
              // onChanged: _onChanged,
              inputType: InputType.both,
              decoration: InputDecoration(
                labelText: ('Time Of Birth').tr(),
              ),
              lastDate: DateTime.now(),
              format: DateFormat("dd-MM-yyyy HH:00"),
              initialValue: DateTime.now(),
              // enabled: true,
            ),```

**Comment by Orlyga01** [comment]
I found out that this is happening when the widget, in which the datepicker is in, is created as part of a setState change, hence widget babyForm, is shown after user clicks next. How should I fix the focus problem, given this situation?

**Comment by grundid** [comment]
I would assume that this will be solved with #1045

**Comment by deandreamatias** [comment]
Due to lack of response, I will close this issue.
If the bug still exists, feel free to open a [new issue](https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/issues/new/choose)
